packages/profile/src/components/trophies/leaderboard.tsx

```diff
@@ -43,15 +42,9 @@ function Row({
   rank: number;
 }) {
   const { username } = useUsername({ address });
-  const location = useLocation();
-
-  const path = useMemo(() => {
-    if (self) return location.pathname;
-    return [...location.pathname.split("/"), address].join("/");
-  }, [location.pathname, address, self]);
 
   return (
-    <Link className="flex" to={path}>
+    <Link className="flex" to="..">
```

Comment:
Here we need to add the address in params if the user click on someone except himself (self)